acstools/tests/test_polarization.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from .. import polarization_tools
+from astropy import units
```

Comment:
I would swap these. Usual order is: stdlib, third-party, local
```suggestion
from astropy import units
from .. import polarization_tools
```